tools/install_prerequisites.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -eu
+
+# Set the CWD to Omicron's source.
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+cd "${SOURCE_DIR}/.."
+
+HOST_OS=$(uname -s)
+if [[ "${HOST_OS}" == "Linux" ]]; then
+  sudo apt-get install libpq-dev
+  sudo apt-get install pkg-config
+elif [[ "${HOST_OS}" == "SunOS" ]]; then
+  need=(
+    'pkg:/package/pkg'
+    'build-essential'
+    'library/postgresql-13'
+    'pkg-config'
+    'brand/omicron1/tools'
+  )
+
+  # Perform updates
+  if (( ${#need[@]} > 0 )); then
```

Comment:
What is this condition checking?

Reply:
I can make it unconditional, but it's just checking that there are more than zero packages to be installed.